test/acceptance/features/bindAppToService.feature

```diff
@@ -756,3 +756,36 @@ Feature: Bind an application to a service
             """
         Then Service Binding "binding-request-cross-ns-service" is ready
         And The application env var "BACKEND_HOST_CROSS_NS_SERVICE" has value "cross.ns.service.stable.example.com"
+
+    @negative
+    Scenario: Service binding with a missing app and configMap as a service
```

Comment:
Would it be possible to binding just a single key from config map? We should have a test for that. https://github.com/redhat-developer/service-binding-operator/issues/389 mentions Secrets as well.. we should check if they could be used as service as well.